imaging/bragg-edge-imaging-2D.ipynb

```diff
@@ -85,24 +73,24 @@
    "outputs": [],
    "source": [
     "def load_component_info(file, advanced_geometry=False):\n",
-    "        from mantid.simpleapi import LoadEmptyInstrument\n",
-    "        ws = LoadEmptyInstrument(Filename=file, StoreInADS=False)\n",
-    "        source_pos, sample_pos = sn.mantid.make_component_info(ws)\n",
-    "        geometry = {}\n",
-    "        geometry[\"source_position\"] = source_pos\n",
-    "        geometry[\"sample_position\"] = sample_pos\n",
-    "        pos, rot, shp = sn.mantid.get_detector_properties(\n",
-    "            ws, source_pos, sample_pos, spectrum_dim='spectrum',advanced_geometry=advanced_geometry)\n",
-    "        x = int(np.sqrt(pos.shape[0]))\n",
-    "        pos2d = sc.fold(pos, dim='spectrum', dims=['y', 'x'], shape=[x, x])\n",
-    "        geometry[\"position\"] = pos2d\n",
-    "        if rot is not None:\n",
-    "            geometry[\"rotation\"] = rot\n",
-    "        if shp is not None:\n",
-    "            geometry[\"shape\"] = shp\n",
-    "        geometry[\"x\"] = pos2d.fields.x['y', 0]\n",
-    "        geometry[\"y\"] = pos2d.fields.y['x', 0]\n",
-    "        return geometry"
+    "    from mantid.simpleapi import LoadEmptyInstrument\n",
```

Comment:
Can you use https://scipp.github.io/scippneutron/generated/scippneutron.load.html#scippneutron.load with the `mantid_alg` parameter? Could that avoid some of the manual steps here?